routers/web/repo/actions/actions.go

```diff
@@ -424,3 +429,94 @@ func decodeNode(node yaml.Node, out any) bool {
 	}
 	return true
 }
+
+func DeleteRuns(ctx *context.Context) {
+	rd := ctx.Req.Body
+	defer rd.Close()
+
+	req := DeleteRunsRequest{}
+	if err := json.NewDecoder(rd).Decode(&req); err != nil {
+		ctx.ServerError("failed to decode request body into delte runs request", err)
+		return
+	}
+	if len(req.ActionIDs) < 1 {
+		ctx.ServerError("missing action_run.id for delete action run", nil)
+		return
+	}
+
+	var (
+		eg               = new(errgroup.Group)
+		actionRuns       []*actions_model.ActionRun
+		jobIDs, taskIDs  []int64
+		taskLogFileNames []string
+	)
+	eg.Go(func() error {
+		var err error
+		actionRuns, err = actions_model.GetRunsByIDsAndTriggerUserID(ctx, req.ActionIDs, ctx.Doer.ID)
```

Comment:
Why use `Doer.ID` here? For example: user A is admin, an ActionRun is not created by them, then user A can't delete that ActionRun?